scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -87,6 +87,37 @@ impl ReplicatorConsumer {
         self.update_or_insert(data, true).await;
     }
 
+    // Returns tuple consisting of TTL, timestamp and vector of consecutive values from primary key.
+    fn initialize_data_from_cdc_row<'a>(
```

Comment:
Refactors such as extracting logic to a separate function deserve their own commit.

Reply:
I'm not sure if I like the name. "[Initialization](https://en.wikipedia.org/wiki/Initialization_(programming))" suggests that it sets some variables or fields to initial values, but to me it looks more like it extracts and returns information from a data structure. This sounds more like a getter.

How about `get_common_cdc_row_data`?